OWML.ModHelper/ModConsole.cs

```diff
@@ -1,56 +1,37 @@
 ﻿using System;
-using System.IO;
-using System.Linq;
-using System.Text;
 using OWML.Common;
 
 namespace OWML.ModHelper
 {
-    public class ModConsole : IModConsole
+    public abstract class ModConsole : IModConsole
```

Comment:
ModOutput?

Reply:
Needs to remain as ModConsole to avoid breaking backwards compatibility.

Reply:
@amazingalek ^ (never sure how email notifications work here)

Reply:
Ah, because of the static event, right.